docs/dev/search-integration.rst

```diff
@@ -117,6 +117,11 @@ Tags to be ignored:
 
 - ``nav``
 
+Special Sphinx and MkDocs heuristics:
+
+- ``.linenos``, ``.lineno`` (line numbers in code-blocks)
```

Comment:
So as I understand, this is pygments generated code? @stsewd I'll add it in a comment and in the docs... let's track the origins of these patterns, we might end up with more...

Reply:
I think they are more from Sphinx and MKDocs material

https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-line-numbers

Not sure if pygments does the line number generation